about.html

```diff
@@ -197,7 +197,7 @@ <h3 class="text-center mt-4 color-text">Partner</h3>
       </div>
     </footer>
```

Comment:
- [ ] Great job working on the desktop version but th footer in your about page is not aligend well since it is very important to fix the alignement and it is one of the main requirements for the porject to make your project looks nicer 🔥♻✨💯💯 so kindly fix it to look like th given template 🚀🙏🏻

- Your design
![image](https://user-images.githubusercontent.com/58553711/211715340-07f43353-c0c6-48b4-b164-c02c459e7b0b.png)

- Given design
![image](https://user-images.githubusercontent.com/58553711/211715420-39d3ae7f-5e2d-4a95-9855-3d98e1b81303.png)